tests/test_gabbits.py

```diff
@@ -48,6 +49,9 @@ def start_fixture(self):
         environ = os.environ.copy()
         environ.update(self.environ)
 
+        # run database migrations (requires ROCKET_DATABASE_URL to be set)
+        alembic.config.main(['upgrade', 'head'])
```

Comment:
Should we also cleanup the database at `stop_fixture()`? Or you plan to add this in separate PR?

Reply:
If you don't mind, I'll do it in a follow up PR. Maybe we should be more cautious and create a temporary database instead of trying to drop the data from the one that `ROCKET_DATABASE_URL` points to (same for unit tests).

Reply:
I don't mind, we can do this. But it's just something we need anyway, otherwise we won't be able to run the same tests multiple times.

Reply:
Yeah, agreed. I wonder if we could keep some subset of tests that we can run against a live XSnippet instance w/o access to the DB.